recipes/mitofinder/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "1.4" %}
+
+package:
+    name: mitofinder
+    version: '{{ version }}'
+
+source:
+    url: https://github.com/RemiAllio/MitoFinder/archive/v{{ version }}.tar.gz
+    sha256: 8b3f6bf9f12d2f993e9886a6fc62f09fc9aa7631324ee787191325e4df5c077a
+
+build:
+  number: 0
+
+requirements:
+    build:
+        - {{ compiler('c') }}
+    host:
+        - python  <3
+    run:
+        - python  <3
+        - biopython
+        - blast
+        - openjdk
+        - pandas
+        - megahit
+        - trnascan-se
+        - spades
+        - idba
+
+test:
+  commands:
+    - mitofinder -h
+
+about:
+    home: https://github.com/parklab/xTea
```

Comment:
@bgruening I just noticed, is this link correct? Shouldn't it be https://github.com/RemiAllio/MitoFinder instead?